ztp/extra-manifests-builder/99-one-shot-time-sync/sync-time-once.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+SYNC_ATTEMPT_TIMEOUT_SEC=300
```

Comment:
Instead of hard-coding this, we could easily allow environment overrides, which would let customers configure this timeout via a simple systemd drop-in, if 300s was not right for their environment:
```suggestion
SYNC_ATTEMPT_TIMEOUT_SEC=${SYNC_ATTEMPT_TIMEOUT_SEC:-300}
```

Reply:
Right, fixing...

Reply:
done